.github/workflows/update-parset-tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Update Test262 parser tests
+env:
+  YARN_ENABLE_SCRIPTS: false # disable post-install scripts
+  YARN_NODE_LINKER: pnp # use pnp linker for better performance
+on:
+  workflow_dispatch:
+    inputs: {}
+  schedule:
+    - cron: "0 0 * * 5"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: tc39/test262
+          path: build/test262
+      - name: Use Node.js latest
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: "*"
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            yarn-
+      - name: Get latest test262 version
+        id: test262
+        run: echo "::set-output name=sha1::$(sh ./scripts/parser-tests/get-test262-version.sh)"
+      - name: Update test262 commit
+        run: |
+          echo ${{ steps.test262.outputs.sha1 }} | ./scripts/parser-tests/bump-test262-version.sh
+      - name: Build babel parser
+        run: |
+          yarn install --immutable --skip-builds
+          yarn gulp build-rollup
+      - name: Update test262 allow list
+        run: |
+          make test-test262-update-allowlist
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
```

Comment:
We can also pass `token: ${{ secrets.BOT_TOKEN }}`, and set `author`/`committer` to @babel-bot.

As an alternative, we could commit by default (in this case it should be pretty safe), similarly to what we do in https://github.com/babel/babel/blob/main/.github/workflows/release.yml#L49-L53

Reply:
I slightly prefer current approach because once two (maybe more in the future) workflow push to `main`, there will be chances when an extra test262 commit is pushed between release tag and the change log commit. On the PR approach, approval of PR can not happen simultaneously with releasing.